src/Lentille/Worker.hs

```diff
@@ -12,11 +12,13 @@ module Lentille.Worker
   ( run,
     getBugzillaSession,
     searchExpr,
+    toTrackerData,
   )
 where
 
 import Control.Monad.Catch (MonadThrow)
 import Data.Aeson (decodeFileStrict)
+import Data.Text as T (null)
```

Comment:
since you are using the qualified version below, it's better to simply import the package qualified, e.g.: `import qualified Data.Text as T`